tests/multi_processing/task_manager.py

```diff
@@ -205,6 +205,7 @@ def testUpdateLatestProcessingTime(self):
     """Tests the _UpdateLatestProcessingTime function."""
     manager = task_manager.TaskManager()
     task = manager.CreateTask(self._TEST_SESSION_IDENTIFIER)
+    task.last_processing_time = 0
```

Comment:
Why this change? is this a left over that could be removed?

Reply:
This is because the UpdateLatestProcessingTime() method compares this field against the latest_processing_time value, and Python3 errors on comparing None and int.

Reply:
Should we update _UpdateLatestProcessingTime instead? I assume this value being None will happen runtime as well

Reply:
You requested that I not change that method ;) https://github.com/log2timeline/plaso/pull/1956/files/6b89d53e2d7c8e7089459e271c518d9e216ed853#diff-dce07e43bc91fcf6c32c3d51d2590951R254

No, this doesn't happen at runtime, because UpdateProcessingTime() is called before this method is invoked.